daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/TypeCalculator.scala

```diff
@@ -176,17 +176,17 @@ class KeysetValueTypeCalculatorUnordered(valueMap: HashMap[DataValuePrimitive, D
     if (valueMap.contains(x)) {
       valueMap.get(x) match {
         case Some(a) => (a, Maybe.Nope)
-        case None => (DataValue.NoValue, One(s"Value ${x} not found in keyset-value mapping"))
+        case None => (DataValue.NoValue, One(s"Value ${x} not found in enumeration dfdlx:repValues"))
```

Comment:
This is unreachable. If valueMap.contains(x), then valueMap.get(x) will not return None. 
You  don't need both an if and a match case here. You valueMap.get(x) returns an Option, so you can just do the match-case.